homeassistant/components/sonos/__init__.py

```diff
@@ -260,8 +259,6 @@ async def _async_create_discovered_player(self, uid, discovered_ip, boot_seqnum)
                     self._create_soco, discovered_ip, SoCoCreationSource.REBOOTED
                 ):
                     async_dispatcher_send(self.hass, f"{SONOS_REBOOTED}-{uid}", soco)
-            else:
-                async_dispatcher_send(self.hass, f"{SONOS_SEEN}-{uid}")
 
```

Comment:
Is there any reason to not accept both activity and discovery so we can reduce the chance we have to poll?

Reply:
I thought I had seen discovery trigger false positives (i.e., when speaker was **not** actually available), but that was probably my poor testing methodology. I think this makes sense and will completely avoid the need to poll for well-working setups. 👍